### PR TITLE
github pull request exercise

### PR DESCRIPTION
**Pull request by heereman**
This pull request is an exercise to fullfill the Software/Data Carpentry instructor training held in May, 2016 in Edinburgh

### CODE REVIEW

File: 00-sql-introduction.md [hunk 1]
```diff
@@ -78,7 +85,15 @@ In particular, we want to use a tool where it's easy to repeat our analysis
...
-Putting our data into a database and using SQL will help us achieve these goals.  
+Putting our data into a relational database and using SQL will help us achieve these goals.  
+
+> ### Definition: _Relational database_
+>
+> A relational database is a digital database where all data is stored in relations (or tables)
+> containing rows and columns. All tables contain records and all records are identified by a
+> field contiaing a unique value. Every table shares at least one field with another table in
+> one-to-one, one-to-many or many-to-many relationships. This allows the user to access the data
+> in many ways
```
Comment (brainwane) [inline 00-sql-introduction.md]: I'm wondering whether there's a way to structure this definition in a way that doesn't give the learner as many pieces of jargon (rows, columns, relations, records, value) in one paragraph. Do you have a concept map we could use to guide us as we structure this?

Should we reuse the word "database" in "digital database" or could we use a phrasing like "data storage application"?

You should probably add a period at the end of the last sentence.

File: 00-sql-introduction.md [hunk 2]
```diff
@@ -78,7 +85,15 @@ In particular, we want to use a tool where it's easy to repeat our analysis
...
 actually modifying our source data.  
 
-Putting our data into a database and using SQL will help us achieve these goals.  
+Putting our data into a relational database and using SQL will help us achieve these goals.  
+
+> ### Definition: _Relational database_
+>
+> A relational database is a digital database where all data is stored in relations (or tables)
+> containing rows and columns. All tables contain records and all records are identified by a
+> field contiaing a unique value. Every table shares at least one field with another table in
```
Comment (brainwane) [inline 00-sql-introduction.md]: Spelling: "containing".

File: 00-sql-introduction.md [hunk 3]
```diff
@@ -3,6 +3,13 @@ layout: lesson
 root: .
 title: Databases using SQL
 minutes: 60
+exercises: 5
+questions:
+- "What is a relational database and why should I use it?"
+- "What is SQL?"
+objectives:
+- "Understand the benefits of using a relational database"
+- "Set up a small database using SQLite"
```
Comment (brainwane) [inline 00-sql-introduction.md]: These look pretty good to me!

File: 00-sql-introduction.md [hunk 4]
```diff
@@ -153,16 +168,19 @@ we downloaded earlier.  Close the currently open database and then
...
+6. Give the table a name that matches the file name (surveys, species, plots), or use the default
+7. If the first row has column headings, check the appropriate box
+8. Make sure the delimiter and quotation options are appropriate for the CSV files.  Ensure 'Ignore trailing Separator/Delimiter' is left *unchecked*.
+9. Click **OK**
+10. When asked if you want to modify the table, click **OK**
+11. Set the data types for each field: choose TEXT for fields with text
    (`species_id`, `genus`, `sex`, etc.) and INT for fields with numbers (`day`,
-   `month`, `year`, `weight`, etc.)
+   `month`, `year`, `weight`, etc.), click **OK*
+12. Click **OK* one more time to confirm the operation.
```
Comment (brainwane) [inline 00-sql-introduction.md]: Glad to have the additional detail.

**Comment by orchid00** [comment]
@brainwane would you like to provide a concept map? I agree less jargon will benefit learners

**Comment by brainwane** [comment]
@orchid00 I'll open up a new issue to ask some questions I have first! But I am open to making a concept map, yes. :)

**Pull request merged**